README.md

```diff
@@ -286,6 +291,42 @@ Users often struggle to find specific books in a large catalog. An advanced sear
 ...
 ```
 
+## Documentation Ticket Introduction
+A **Documentation Ticket** is a focused, ongoing ticket written as GitHub Issues dedicated to project documentation.  Unlike development-focused tickets, Documentation Ticket remain open continuously, evolving as updates are needed, and can be reopened or revised indefinitely. They are not directly tied to Pull Requests (PRs) but can be referenced for context.
```

Comment:
What is the reason to bring up a new term of ticket. If it has not been used in the project yet. There is written in the first sentence, that it is GH Issue. Why we don't have Documentation Issue? It is starting to be confusing for a newcomer with having: features, tickets, issues: all pointing to GH Issue (feature issue, documentation issue, ordinary issue).

Reply:
Confusion was observed from the first demo and related talks about Living documentation.
- Documentation issues in GitHub include the implementation of documentation tickets (types: user story, feature, or requirement).
- I tried to sync all docs to describe the Documentation ticker as an abstract concept with GH Issue implementation.